code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,66 @@
+package code401.challenges.linkedlist;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class LinkedList {
+    Node head;
```

Comment:
You should get in the habit of declaring instance variable as either private or public
`public Node head;`
`public int size;`